dwin_quiz/src/component/constantly-page/menu-block/Menu-block.scss

```diff
@@ -34,14 +34,29 @@
     svg {
         margin-right: 10px;
     }
+    .activeButton {
+        background-color: #696f79;
+        color: #f0e9e9;
+        margin-bottom: 4%;
+        width: 185px;
+        border-radius: 30px;
+        height: 45px;
+        border: 0;
+        font-weight: 600;
+        cursor: pointer;
+
+        .imgDashBoard * {
+            fill: #f0e9e9;
+        }
+    }
```

Comment:
Why you need to use different classNames if buttons have the same styles and can have same classNames?